release-notes/topics/operator-220.adoc

```diff
@@ -0,0 +1,14 @@
+[[operator-220-intro]]
+= Automation Platform Operator 2.2
+
+The Ansible Automation Platform Operator provides cloud-native, push-button deployment of new Ansible Automation Platform instances in your OpenShift environment.
+
+.Deprecated functionality
+
+* The `image_pull_secret` (string) variable has been deprecated and will no longer be supported in a future release. You should now use the `image_pull_secrets` (array) variable on the spec when creating an Automation Hub or Automation Controller custom resource. This new variable allows you to specify multiple pull secrets as an array, for example:
```

Comment:
Same suggestion as above. Use attributes.
```suggestion
* The `image_pull_secret` (string) variable has been deprecated and will no longer be supported in a future release. You should now use the `image_pull_secrets` (array) variable on the spec when creating an {HubName} or {ControllerName} custom resource. This new variable allows you to specify multiple pull secrets as an array, for example:
```

Reply:
Leaving for now.